mcp3008.js

```diff
@@ -27,6 +27,12 @@ function read(channel, callback) {
     });
 }
 
+function stopInstance (instance) {
+    if (instance != undefined) {
```

Comment:
Please use `!==`.

Reply:
Sure, but `!=` was written by you in deca1530db2de244b46ec589cabde6fbf4c9e41f I was just returning the code.